packages/augur-ui/src/modules/market/components/market-orders-positions-table/open-orders-table.styles.less

```diff
@@ -2,6 +2,18 @@
 
 .Tabs {
   cursor: default;
+
+  @media @breakpoint-mobile {
+    > div[class*="Content"] > div.Table {
+      > ul[class*="DataTableHeader"] > li:nth-of-type(3) {
```

Comment:
Can you just add the style in DataTableHeader.styles.less and Row.styles.less?

Open orders is .Row for that one. 

Shouldn't be this complicated

Reply:
agreed, will update